inspire/dojson/hep/fields/bd1xx.py

```diff
@@ -24,6 +24,7 @@
 
 from dojson import utils
 
+from inspire.dojson import utils as utils_
```

Comment:
Maybe you can rename this to `inspire_dojson_utils` so that it's clear what they are about.